### PR TITLE
 MAILBOX-366 Events should carry an EventId 

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: mailbox/store/src/main/java/org/apache/james/mailbox/store/event/EventFactory.java [hunk 1]
```diff
@@ -48,27 +49,29 @@
     }
 
     public MailboxListener.Added added(MailboxSession.SessionId sessionId, User user, SortedMap<MessageUid, MessageMetaData> uids, Mailbox mailbox) {
-        return new MailboxListener.Added(sessionId, user, mailbox.generateAssociatedPath(), mailbox.getMailboxId(), uids);
+        return new MailboxListener.Added(sessionId, user, mailbox.generateAssociatedPath(), mailbox.getMailboxId(),
```
[the remark this answers is not in the document]
Comment (chibenwa) [reply mailbox/store/src/main/java/org/apache/james/mailbox/store/event/EventFactory.java]: Keep in mind that we soon ( https://github.com/linagora/james-project/pull/2058 ) gonna have some builders.

The decision do not lie in the constructor but rather in these builders

**Comment by chibenwa** [comment]
Was :green_apple: 

test this please

**Comment by aduprat** [comment]
Was :green_apple: before restarting the CI.

**Comment by openpaas-james** [comment]
Build triggered. sha1 is merged.

**Comment by openpaas-james** [comment]
Build started sha1 is merged.

**Comment by chibenwa** [comment]
test this please

**Comment by trantienduchn** [comment]
```
[60919d8591622d2f53365cdb2d2db5138e6a7065] [ERROR] Errors: 
[60919d8591622d2f53365cdb2d2db5138e6a7065] [ERROR]   CassandraEventStoreTest » ExceptionInInitializer
[60919d8591622d2f53365cdb2d2db5138e6a7065] [INFO] 
[60919d8591622d2f53365cdb2d2db5138e6a7065] [ERROR] Tests run: 16, Failures: 0, Errors: 1, Skipped: 0
```
test this please, timemout

**Pull request merged**